lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
```

Comment:
I guess it's a question of effort vs accuracy, but if the intent is to get results similar to classic, we should probably have a small corpus of files for each language, get the overall mass of each file as calculated both by classic & by this engine, and compare those to see if the thresholds (and to a lesser extent `per_point` values) need to be scaled to get matching results.